domain/src/main/kotlin/no/nav/su/se/bakover/domain/klage/AvvistKlage.kt

```diff
@@ -115,6 +115,8 @@ data class AvvistKlage(
             saksbehandlerNavn = hentNavnForNavIdent(this.saksbehandler).getOrElse {
                 return KunneIkkeLageBrevRequestForKlage.FeilVedHentingAvSaksbehandlernavn(it).left()
             },
+            attestantNavn = this.attesteringer.prøvHentSisteAttestering()?.attestant?.let { hentNavnForNavIdent(it) }
```

Comment:
Savner en funksjon ala. `this.sisteAttestant(): String?`

Reply:
Denne lagBrevRequest-funksjonen kalles i tre tilfeller:
- oversendelse (da ligger attestant allerede på objektet)
- avvisning (da ligger attestant allerede på objektet)
- brevutkast (da er det ikke sikkert attestanten ligger på objektet, men den kan være underkjent og da vil den ligge her)

Så i de 2 første tilfellene vil vi jo kreve at det finnes en Attestering.Iverksatt, hvis ikke har vi jo noe internfeil. 
Men hva ønsker vi å vise for brevutkastet til saksbehandler og attestant? Dersom det ikke finnes en attestant enda, har vi brukt å vise "-" i brevet. Dersom det er attestant som genererer brevutkastet, viser vi bare attestanten som gjorde http-requesten. Og i det tilfellet den tidligere er underkjent, og en saksbehandler skal vise brevet på nytt, må vi velge mellom "-" og evt. den attestanten som underkjente.

Jeg foreslår og splitte denne funksjonen opp i flere. Siden den er for generell til å avgjøre attestant selv, så den bør ta det inn, men samtidig ønsker vi ikke at servicen skal bestemme attestant, siden dette er domenelogikk. En mulighet er å flytte denne funksjonen i bunn av fila og heller ha den som en internal extension-function, som er reservert for domenet (må nøye oss med kdoc). Deretter har du spesifikke funksjoner per data class, noen gir mulighet til å generere utkast, mens andre gjør ikke det.